cmd/watch.go

```diff
@@ -144,14 +131,33 @@ func activateDirWatcher(targetDir string) {
 				// all events are caught by default
 				log.Printf("event: %v, eventT: %T", event, event)
 				if event.Op&fsnotify.Write == fsnotify.Write {
-					esrc := EventSrc(event)
-					ev, err := esrc()
+					fsEv := &FsEvent{
+						event: event,
+						eventPath: func(evPath string) (string, error) {
```

Comment:
In unit tests, you can overwrite this function to a function which doesn't call the `os` package.